libs/core/src/lib/localizator-editor/localization-editor-main/localization-editor-main.component.html

```diff
@@ -1,7 +1,10 @@
 <fd-input-group [compact]="compact"
+                [state]="state"
```

Comment:
Why is there a state on the input group and the addon?

Reply:
Otherwise it's not applied to inputs.